### PR TITLE
Pep fixed the whole codebase

### PR DESCRIPTION
**Pull request by Blatzar**
using `autopep8 --recursive -i --aggressive anime-downloader/anime_downloader/`

### CODE REVIEW

File: anime_downloader/sites/animixplay.py [hunk 1]
```diff
@@ -62,15 +62,15 @@ def _scrape_episodes(self):
             # In extremely rare cases the anime isn't loaded and must be generated by the server first
             try:
                 data = (helpers.post('https://animixplay.com/raw/2ENCwGVubdvzrQ2eu4hBH',
-                        data={data_id: post_id}).json())
+                                     data={data_id: post_id}).json())
             # 400 HTTPError here
-            except:
+            except BaseException:
```
Comment (Blatzar) [inline anime_downloader/sites/animixplay.py:67]: TODO: Look into if this causes issues

File: anime_downloader/sites/helpers/selescrape.py [hunk 2]
```diff
@@ -88,12 +88,12 @@ def driver_select(): #
...
         fireFoxOptions.headless = True
         fireFoxOptions.add_argument('--log fatal')
-        if binary == None:  
+        if binary is None:
             driver = webdriver.Firefox(options=fireFoxOptions, service_log_path=os.path.devnull)
         else:
             try:
                 driver = webdriver.Firefox(options=fireFoxOptions, service_log_path=os.path.devnull)
-            except:
+            except BaseException:
```
Comment (Blatzar) [inline anime_downloader/sites/helpers/selescrape.py:96]: Another BaseException I'm unsure works, noted for later

File: anime_downloader/sites/animefreak.py [hunk 3]
```diff
@@ -6,42 +6,42 @@
 
 
 class AnimeFreak(Anime, sitename='animefreak'):
```
Comment (IguanasInPyjamas) [inline anime_downloader/sites/animefreak.py:8]: oof all those tabs

File: anime_downloader/sites/animekisa.py [hunk 4]
```diff
@@ -7,53 +7,56 @@
 
 logger = logging.getLogger(__name__)
 
-class AnimeKisa(Anime,sitename='animekisa'):
```
Comment (IguanasInPyjamas) [inline anime_downloader/sites/animekisa.py:10]: feelssadman

**Review by IguanasInPyjamas — APPROVED** [review]
So pretty

**Pull request merged**